lib/services/ntpd.pm

```diff
@@ -58,7 +59,10 @@ sub check_function {
     assert_script_run("date -s 'Tue Jul 03 10:42:42 2018'");
     assert_script_run("date | grep 2018");
     systemctl("restart ntpd.service");
-    script_retry("date | grep -v 2018", delay => 30, retry => 6);
+    assert_script_run("echo '* * * * * root date' > /etc/cron.d/ntptest");
```

Comment:
Why cron ? it will repeat just every minute, so with the timeouts I see below it will prolong test run of this simple test a lot